### ISSUE TITLE
Host review status badge at pyopensci.org

### ISSUE THREAD

**Issue by sneakers-the-rat** [pyOpenSci/software-peer-review#293]
Finishing up a review now, and i see that the badge uses `tinyurl`: https://github.com/pyOpenSci/software-peer-review/blob/a603e60f50511667b14d32809f2e44678ec3c1f0/appendices/package-approval-template.md?plain=1#L12

that resolves to this big old thing:

<details>
<summary>expand/collapse link</summary>

https://img.shields.io/badge/PyOpenSci-Peer%20Reviewed-success.svg?logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHIAAABnCAYAAADcxhZVAAAACXBIWXMAAAWJAAAFiQFtaJ36AAAHXUlEQVR4nO1dy3EbORBtq3SX9oiTtBGIG4HoCCxHYG4EpiIQFYGpCExFYCoCUxEsGcFyTziuGIG2wHpjDrnzwTQaMwMMXhWrVJI4vzfofzc+vL+/k29orcdEdI2PwbjilGsieiOirflZKbX2foERwAuRWmtD2B0+twKHfCGipfkopd4EjhcdRInUWl8S0ZyIvnh6UDscf54IPYYYkVrrERGtiOhC7vJKYQgdJ7F7wJnEQVomkXCeFc47eJAUkRB3pyT+Q0RPRPTs6Umb8y0hzgcPZ9GKB/nvya83EH1vdFixf3l62H8qpRaejh0MJFZkkXib5Y0R6DJfK9NYxoOHlGg9xargd1tP5xq8aCWPRE4KflcVBEhwhC8iv2mtp4TggNZ6IRQYSCjBuccHY8j8lh58O/C1IhMSkQkcJCIjgYSOND7iR4fvGz806VJHOBMJx7/Ib7SC1lrkRhBhKou9bpVSvvzYXqCUSDyYO/h/1x4vVsqhNyT+LPuj1trEfhexpsAKidRaz4ho2mI2ow1cEdGDCVZore9iS4EdGTtmFWqtV7jhmEjM4yrGFNip1bocSATmAmI2GvwiEuJ0SGG0G9xzFNgTiWKpaSw31QDTWBLT2YqMzbCxxUUs+cyMyKK001AQB5EoHg55NWaOPtfX/SR4LZ3hLIKEbxZVYgct8DIHjdCJ3MFlIsf78Bm5agVngde87MNtsDxdXKcoiLzpwXVw8KyUyvxAV9cpCiJDxKNSam9pI9Q2eCJ91uxIYwPDZp6lpCBSTahtiD7wEXwQuYEBws5RnuCtKFNhMhiJxAMkiTQETnylh3IuwhhOfKi63QukiHzOdBYd9NYEDz098BYgQeQmZ3hkOiuKaElIkCAyqyi/hF4McQUGX89zhj5GLl6VUplRswxYjFoTiSqKpcW/topz3MQV86QL3Nwk8KS0FZF9ljpnqEvlIvtu6Kmg2mfQd9XhRGTO1QjZuNnVuUywwtd9Vh3n0G3fe3AtXaFS35UNujip91mVBS7awn6GAEogG+s4pdQHOOqlhcEB4GPOYLMisQZZKHGllGrNKMqC5kMdpvBaQeIEAyyahgCN+P1KRD+01ibFNkdxm1fsicRUDBc3JFQUZk1AooS6uQCpfxup57MSIZ/GGloB1n1JMH7qyWYwquunacP3sUJ/EQkRcy99gp7iSSk1P700IwZbaPEzc/rWWPViOEos4+ZiJtOoj89KqbJEdFtWpxG537E6RUptCidfwVqb1fmHglbrq+P367CFFVlr1AnqR1scTQnjonKEWU3z6F4cSxBpXgiX70ujAzKNpHBq9ZOYRSexIl1a17044h1UIJjSzhG3s7ovRLrA+IJezPoOxpeyxWya6lEBrPQxVsspXj343jd1IcMyJCJrUEamkQJKqWvo9z9g7W8ETnnL6dtMRFqgZmXu/25cN6XUCPre1Qp/aNoaHwKRj0T0G976l64uoo7M3P+toLPv6/63Bv8LWFSh70SaivL9EF+89XdC4osFkDmyuQYEV2qJr8Btk+hP34kscuA7rZeBe2BlJduu4gpY60pnIsvSQEIoCl913j3WxD1wJPPKdlVKrUhf4m6ej0XmCp+DAsjkThCxul+RDVwQBfnhfKBiZKPHLnFTp865SUc1Mgy6ArcSg4h+r4v4SO7Es/C4lVIZNjD5g4BDFKz2ZRUzdtA28CR1PAtsQmubhz3B8TFrxauo1Yo8n4RDXIUd3JJRoFMeOWrgpi5v6W3/SIiRrCOLW8meRzRbD5qiLEYgvrTaj3x2LOOk+xOjRmWET2WOE9jmP55dnC6wYhR1j6uah1tpPYfFte3ame8ROERWitYUNO8GnER4pRRLRHYD8X7MRGQH8DEoP6TxLKVAxKQJ1hUlkUEiCiIZYa/otiocqmjttM/Rx2D8WIgMrQFJXCLEQmRj46HjGa2cc1faAbEQKe6XeQaHyMqXdbArsqvMicNs2cqXNRYiObHYTx1tFcGagFLXFhEFkbhJjsHTRdkIx3+tLQONyf3grMpWgwIwsDiuT22yISYiOQMtrrLd2VsCd+jGcIhEzpIjXmdt6Er0c3AS7M82ifTYIjucksML33lSiNQH5tetVnFsRC6ZhcC3qAIUB8Jx3BeldA7QKaIiEiKIq/O+SJMp0ChrLWG8FV91Ca21ywDAV8xmd8oZwohyGfVyNF68DrESOcL4MS52KFtsvLE29KHrpqrm/NdNzh0lkXSwErkGRoYdjI1lla5CleAYYl0iRfa56UDCaIkkt16LMpjq9tNVMhIeFvGY2yrKGrETGdrA/EZ6MY+oiaSwyHRqSIq+1AMGw7jLlnULvLim1aJfkRl6vDLZ4jSPwRCZAaM8v/bjauSadAdHJB18vYVQlxgH4hvCDZJIOojaqYCv2QTGL535aJUfLJEZ4MzPUILha3ggO1Jki8ETmQErdIKPlEH0gqiQ910cEpEFyIXcxojc2BKb7fmxbruzOhFpCQTiyyoJtj46rJogERkJUn9kJEhExgAi+g/1Qv7cFT6QFQAAAABJRU5ErkJggg==

</details>

and the length of that is (i assume) why we use the tinyurl. 

The problem with the tinyurl thing is that it's a) super fragile - the tinyurl can change at any time and so then we just have dead links. same thing with the shields.io API and also b) it puts effectively a tracking link in the readmes of all our packages.

It would be super simple to just make the badge available at, eg. https://pyopensci.org/badges/peer-reviewed.svg

going to follow up with PRs to this effect but wanted to open an issue to track them.

PRs:
- [x] https://github.com/pyOpenSci/pyopensci.github.io/pull/428
- [x] https://github.com/pyOpenSci/software-peer-review/pull/294
- [x] https://github.com/pyOpenSci/python-package-guide/pull/318
- [x] https://github.com/pyOpenSci/pyosPackage/pull/37

**Comment by lwasser** [comment]
@sneakers-the-rat i think i've now merged everything needed for this issue. Is our next step to reach out to maintainers and ask them to use this link instead in their readme?

**Comment by sneakers-the-rat** [comment]
Do we want to wait on that until we figure out how to do package-specific badges? I figure one ask will be less of a pain than two.

There's one more way to do dynamic badging without needing to generate a zillion of the same svg file using htaccess files I want to try too btw, forgot github respects those

**Comment by sneakers-the-rat** [comment]
Or wait, maybe thats only enterprise accounts? I seen it working over here: https://github.com/perma-id/w3id.org/blob/master/linkml/.htaccess

I also forgot about HTTP-REFRESH meta fields
https://github.com/jekyll/jekyll-redirect-from

**Comment by lwasser** [comment]
oh right - i'm just getting caught back up and i forgot about that part. Sure -- let's think about package specific badges. I like your idea of hacking jekyll to do it 😆 especially if it's just a matter of parsing the YAML file (which will eventually be a database), looking at things like last commit date, peer review status, etc. then a cron job that runs weekly or something to that effect.

**Comment by lwasser** [comment]
how would redirect from work? i've used that for page redirects but that just adds a refresh to the page meta (if i'm remembering correctly that is)

**Comment by sneakers-the-rat** [comment]
yeah right, i was just thinking maybe github's image resolver would maybe understand the refresh tag where it didn't understand the javascript? let me test real quick

![this should be an image](https://hackers.piracy.solutions/dynamic_img/my_pkg/)

**Comment by lwasser** [comment]
sounds good!!

**Comment by sneakers-the-rat** [comment]
this is done!
